airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -26,6 +26,8 @@ Breaking changes
 ~~~~~~~~~~~~~~~~
 
 * ``Simplify KubernetesPodOperator (#19572)``
+* Parameter ``is_delete_operator_pod`` default is changed to ``True``
```

Comment:
Can you add the rationale on why we are making this change for users to understand it better

Reply:
done, ptal